Sources/AcmeSwift/APIs/AcmeSwift+Orders.swift

```diff
@@ -329,14 +329,17 @@ extension AcmeSwift {
                 throw AcmeError.mustBeAuthenticated("\(AcmeSwift.self).init() must be called with an \(AccountCredentials.self)")
             }
             
-            let publicKey = login.key.publicKey.rawRepresentation
-            
+            let pubKey = try JWTKit.ECDSAKey.public(pem: login.key.publicKey.pemRepresentation)
+            guard let parameters = pubKey.parameters else {
+                throw AcmeError.invalidKeyError("Public key parameters are nil")
+            }
+
             let jwk = JWK.ecdsa(
-                nil,
+                .es256,
                 identifier: nil,
-                x: publicKey.prefix(upTo: publicKey.count/2).base64EncodedString(),
-                y: publicKey.suffix(from: publicKey.count/2).base64EncodedString(),
-                curve: nil
+                x: parameters.x,
+                y: parameters.y,
+                curve: .p256
```

Comment:
Shoot, I forgot to specify the curve type… You should be able to leave the algorithm set to nil since you weren’t specifying one previously. As for the parameters not matching, seems like I forgot to turn them into URL base64 strings, sorry about that 😅 (no need to import into JWT’s ECDSAKey first, it’s basically doing the same thing)

Reply:
Fixed this in #15, sorry about that!